domain-xample/xample-workflows/src/main/java/gov/va/vro/routes/xample/DbHelper.java

```diff
@@ -81,7 +81,9 @@ public ContentionEventEntity saveContentionEvent(final BieMessagePayload bieMess
     contentionEventEntity.setActionResultName(bieMessagePayload.getActionResultName());
     contentionEventEntity.setDescription(bieMessagePayload.getDescription());
 
-    return contentionEventRepository.save(contentionEventEntity);
+    contentionEventRepository.save(contentionEventEntity);
+    log.info("Saved bieMessagePayload {}", bieMessagePayload);
```

Comment:
@lisac you raise an excellent point here (two of them actually):
1. first, despite this code existing in the xample project, it is a relevant point because xample contains both example/test code and production code. We've talked about prioritizing the separation a long time ago, however I don't see anything in the backlog about it when looking just now.
2. second, I can see that some thought was applied around that concern based on the [NoLogging annotation](https://github.com/department-of-veterans-affairs/abd-vro/blob/a4f75bd8a956dde965b121ff4caaa1fea4be06c5/shared/lib-bie-kafka/src/main/java/gov/va/vro/model/biekafka/BieMessagePayload.java#L45). However, I am not seeing how that annotation would prevent the logging from working in this case since the entity is simply being formatted 

So all that to say that we need to confirm that this log.info call is properly obfuscated.

Reply:
@lisac @nelsestu, I introduced the `@NoLogging` annotation to be used on top of the description field. Then, I incorporated the logic to not print the value of any field that has this annotation in the toString() method.
The benefits of that approach are that it makes the code clean and self-explanatory without introducing additional if conditions.